### ISSUE TITLE
Alcohol abuse, Counseling treatment, Psychiatric treatment flipping to "YES"

### ISSUE THREAD

**Issue by ryanhofdotgov** [truetandem/e-QIP-prototype#2883]
Under 0.5.1 (Sprint 35), Chrome, macOS:

* starting from empty SF-86, quickly tried to fill-out bare bones application, saying NO to all the "extra" questions.
* When I got to review stage for a section, eApp switched all my NO answers to YES, saying I failed validation and needed to provide more info for the alcohol, counseling treatment, psychiatric treatment sections.

**Comment by bmallred** [comment]
@ryanhofdotgov what are the "extra" questions? Are those the final "yes"/"no" blocks at then end of an accordion?

**Comment by bmallred** [comment]
What I have found:

 1. There is an error within the validator logic `alcoholorderedcounseling.js` on line 33

I didn't see the switching values from **no** to **yes** however.

**Comment by ryanhofdotgov** [comment]
Sorry – by "extra" questions I was referring to the alcohol, counseling, etc. questions – they were "extra" from the sense I was trying to submit a "bare bones" application (i.e., one job for entire life, no travel, no "extra" interesting/complicating aspects to application) – the shortest application possible.

I went and clicked No through all these different questions, then when I got to the end of the section it told me I was missing all this extra form data, and showed that I had clicked "Yes" to alcohol issue, etc.
